examples/http_server/client/client

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+curl localhost:8080
```

Comment:
I don't think any of our examples should use non-secure connections. The HTTP server should use TLS, and the local CA cert can be passed to curl using the `--cacert ` command line argument.

Reply:
I have not added TLS to the HTTP server node yet. I will fix this, when TLS support is added.